packages/wrangler/src/pipelines/index.ts

```diff
@@ -101,20 +101,23 @@ function addCreateAndUpdateOptions(yargs: Argv<CommonYargsOptions>) {
 		})
 		.option("batch-max-mb", {
 			describe:
-				"The approximate maximum size of a batch before flush in megabytes \nDefault: 10",
+				"The approximate maximum size (in megabytes) for each batch before flushing (range: 1 - 100)",
 			type: "number",
+			default: 10,
```

Comment:
I see that we mentioned the default value in the description already. Is this the same default value we have on the API side? That's why this is a non-breaking change?

Reply:
Yeah, 10 MB is the current default on the API side. Regarding breaking changes, this is currently in private beta and changes like this are expected.

Reply:
The defaults are still being determined, so I removed here and will instead only set them on the API side.